.github/workflows/integration-test.yml

```diff
@@ -551,7 +554,7 @@ jobs:
 
   EC2LinuxIntegrationTest:
     needs: [MakeBinary, StartLocalStack, GenerateTestMatrix]
-    name: 'Test'
```

Comment:
This so on the actions ui we can see more of the test info

Reply:
This is renamed as "EC2LinuxIntegrationTest" for easier understanding.